configuration/multimedia.md

```diff
@@ -64,24 +64,27 @@ mactts:Kanya Kanya (th_TH)
 
 You can define a default TTS service and a default voice to use either by textual configuration in `conf/services/runtime.cfg` or in the Paper UI in `Configuration->System->Voice`.
 
-In order to say a text, you can enter
+In order to say a text, you can enter such a command on the console or you can enter such commands within DSL rules:
 
-```
+```text
 openhab> smarthome:voice say Hello world!
 ```
 
-on the console or
-
-```
+```java
 say("Hello world!")
+say("Hello world!", "voicerss:enGB")
+say("Hello world!", "voicerss:enUS", "sonos:PLAY5:kitchen")
 ```
 
-within DSL rules.
+The default voice and default audio sink will be used.
```

Comment:
Oh :disappointed: I'm sorry, besides Splitting lines, that's probably the only thing I "improved". My fault, will change back to the way you originally phrased it.